src/scripts/main.js

```diff
@@ -2,4 +2,21 @@
 
 document.addEventListener('click', e => {
   // write code here
+  const wall = document.querySelector('.wall');
+  const spider = document.querySelector('.spider');
+  const spiderSize = spider.clientWidth;
+  const wallSize = wall.clientWidth;
+
+  if (e.target === wall) {
+    let leftPos = e.offsetX - spiderSize / 2;
+    let topPos = e.offsetY - spiderSize / 2;
+
+    leftPos = leftPos < 0 ? 0 : leftPos;
+    leftPos = leftPos > wallSize - spiderSize ? wallSize - spiderSize : leftPos;
+    topPos = topPos < 0 ? 0 : topPos;
+    topPos = topPos > wallSize - spiderSize ? wallSize - spiderSize : topPos;
```

Comment:
1) don't use ternary operator here, code is hard to understand
2) move the calculation of the coords into a separate function (also avoid duplicate code)